src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj

```diff
@@ -5,8 +5,8 @@
     <EnableDefaultItems>true</EnableDefaultItems>
     <IsPackable>true</IsPackable>
     <EnableAOTAnalyzer>true</EnableAOTAnalyzer>
-    <ServicingVersion>2</ServicingVersion>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <ServicingVersion>3</ServicingVersion>
```

Comment:
@ViktorHofer @carlossanlop @ericstj could you please help review this package authoring change in this file? Just to ensure I am doing the right thing here.

Reply:
Looks good.